.eslintrc.yml

```diff
@@ -5,4 +5,6 @@ extends: standard-with-typescript
 parserOptions:
   ecmaVersion: latest
```

Comment:
Don't we want to be explicit here? Especially considering we have `ES2020` in typescript config under `lib`?

Reply:
Correct, we have to use `ES2020` when we lint the code as well.